dojo/problem/helper.py

```diff
@@ -0,0 +1,137 @@
+import json
+import os
+import requests
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
+
+from dojo.models import Problem, Finding
+
+import logging
+logger = logging.getLogger(__name__)
+
+CONFIG_FILE = os.path.join(os.path.dirname(__file__), 'config.json')
+CACHED_JSON_FILE = os.path.join('/app/media', 'cached_disambiguator.json')
```

Comment:
Is `/app/media` always available? This seems like something that would exist in a container, but not on a classic install. Maybe there is a configuration variable we can use to know where to store the cached file.

Reply:
I chose to use `/app/media` because this path is defined in `docker-compose` for the `uwsgi` and `celeryworker` containers using the following volume configuration:
```
volumes:
        - type: bind
          source: ./docker/extra_settings
          target: /app/docker/extra_settings
        - "defectdojo_media:${DD_MEDIA_ROOT:-/app/media}"
```
but really have a problem to choose where is the json if is defined DD_MEDIA_ROOT. I think in change to use:
```
MEDIA_ROOT = os.getenv('DD_MEDIA_ROOT', '/app/media')
CACHED_JSON_FILE = os.path.join(MEDIA_ROOT, 'cached_disambiguator.json')
```